scan/scan.go

```diff
@@ -329,22 +329,31 @@ func lexSpace(l *Scanner) stateFn {
 // If the input base is greater than 10, some identifiers
 // are actually numbers. We handle this here.
 func lexIdentifier(l *Scanner) stateFn {
+	for l.peek() == '@' {
+		l.next()
+	}
 	for isAlphaNumeric(l.peek()) {
 		l.next()
 	}
+	for l.peek() == '@' {
+		l.next()
+	}
 	if !l.atTerminator() {
 		return l.errorf("bad character %#U", l.next())
 	}
 	// Some identifiers are operators.
 	word := l.input[l.start:l.pos]
+	w := strings.Trim(word, "@")
```

Comment:
you know where it is, you could compute it as you go through the @s  at line 335 and 338.
i suppose it's not important as lexOperator doesn't have that luxury. your choice.

in any case both here and below "w" is a lousy name. maybe "word" and "opWord"

Reply:
Fixed; now there's just word, and it's trimmed.